linux-webserver.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# A script to build a webserver on linux
+
+if (multipass version)
+then
+  echo "multipass already intall"
+else
+  echo "install multipass"
+  sudo apt install -y multipass
```

Comment:
Need to use Snap to install multipass, not apt.

https://multipass.run/install